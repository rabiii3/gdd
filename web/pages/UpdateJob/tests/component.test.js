import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore  from '../../../store/configureStore.test';
import { UpdateJobDescription } from '..';

describe( 'app | pages | UpdateJobDescription | component', () => {
  it('should match snapshot', () => {
    const state = {
      jobDescription:{
        data:{
          '0':{
            _id: '0',
            title: 'required js dev',
            content: 'some content',
            skills: ['5d035ef378c332001b67bada', '5d035ef378c332001b67badb'],
            createdBy: '456',
            createdAt: '',
            updatedBy: '456',
            updatedAt: '',
            tenantId: '333'
          }
        },
        sort:{
          sortBy: 'TITLE',
          direction: 'desc',
        }
      },
    };
    const props = {
      classes:{},
      jobDescription:{},
      isEditMode: true,
      setEditMode: jest.fn(),
    };
    const store = configureStore(state);
      const wrapper = shallow(
        <Provider store={store}>
          <UpdateJobDescription {...props} />
        </Provider>,
      );
      expect(wrapper).toMatchSnapshot();
  });
});