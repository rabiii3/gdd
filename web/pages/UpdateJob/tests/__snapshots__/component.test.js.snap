// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app | pages | UpdateJobDescription | component should match snapshot 1`] = `
<Component
  classes={Object {}}
  isEditMode={true}
  jobDescription={Object {}}
  setEditMode={[MockFunction]}
/>
`;
